### DISCUSSION TITLE
Visit Profiles

### DISCUSSION THREAD

**Discussion by Ahm3dsay3d** [felladrin/linkedin-autoconnect-chrome-extension#7] [Ideas]
Add the feature where you can visit profiles LinkedIn 1st connections' profiles.

**Comment by felladrin** [comment]
Hi @Ahm3dsay3d. Thanks for the suggestion.
Would you mind giving more details about it?
What are the advantages of visiting 1st connections' profiles?
Have you seen this feature in some other tool, for me to have a better picture?

**Comment by Ahm3dsay3d** [comment]
Visiting profiles makes your connections visit yours as well. so that they know you still exist and they can see your updates in the market.
And also please add the feature that when you visit them, you can endorse their first skill as well so that you contribute to their profile as well.
Yes, I saw it in professional plugins (for instance phantombuster). but we use it on a personal scale.
Hope you could understand me.
